_posts/2017-08-18-forbes.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Forbes
+sourceURL: https://digiday.com/media/new-mobile-site-forbes-boosted-impressions-per-session-10-percent/
+image: /images/forbes.png
+tags:
+- News
+- Notifications
+- Offline
+- Service Worker
+---
+
+**Forbes’** PWA loads in **2.5** seconds on mobile compared to 6.5 seconds for its previous site. Impressions per visit are up **10%**.
```

Comment:
If I were you, I would...

1. Close this PR.
2. `git checkout master`
3. `git pull master`
4. `git checkout -b feat-social_meta_take2`
5. Make changes to `head.html`, referring to closed PR as needed
6. `git commit -am "Add social sharing meta"`
7. `git push -u origin feat-social_meta_take2`
8. Make a new PR with this branch

If you _want_ to spend time troubleshooting this branch, that's your call, but the changes you're introducing seem so small in comparison to the changes you're _undoing_, it would seem easier to me to just start with a clean slate.